src/containers/EditEventPage.js

```diff
@@ -20,8 +21,14 @@ class EditEventPage extends React.Component {
   render() {
     return (
       <>
-        <EventForm event={this.props.event} onSubmit={this.onSubmit} />
-        <button onClick={this.onRemoveClick}>Remove event</button>
+        {this.props.event ? (
+          <>
```

Comment:
And you probably don't need these inner fragments too 🤔

Reply:
I need this one ;)

Reply:
Well, then maybe you don't need the external one :)